packages/fx-core/src/component/generator/copilotPlugin/generator.ts

```diff
@@ -389,6 +388,7 @@ export class CopilotPluginGenerator {
       context.telemetryReporter.sendTelemetryEvent(specParserGenerateResultTelemetryEvent, {
         [telemetryProperties.generateType]: type.toString(),
         [specParserGenerateResultAllSuccessTelemetryProperty]: generateResult.allSuccess.toString(),
+        [specParserGenerateAuthTypeTelemetryProperty]: authData?.authName ?? "None",
         [specParserGenerateResultWarningsTelemetryProperty]: generateResult.warnings
```

Comment:
How about putting this telemetry event in line 299 so we can have such info in telemetry whenever generator succeeds or fails?
context.telemetryReporter.sendTelemetryEvent(copilotPluginExistingApiSpecUrlTelemetryEvent, {
        [telemetryProperties.isRemoteUrlTelemetryProperty]: isValidHttpUrl(url).toString(),
        [telemetryProperties.generateType]: type.toString(),
      });

Reply:
Sure, thanks for the tip